pages/_document.tsx

```diff
@@ -4,7 +4,7 @@ export default function Document() {
   return (
     <Html lang="fi">
       <Head />
-      <body className="bg-gray-300">
+      <body className="bg-white">
```

Comment:
Olisiko tämän hyvä olla erivärinen kuin "tavallinen" taustaväri?

Reply:
Annetaan olla näin kun teit nyt. Ulla tekee pian designiin visuaalisia parannuksia, niin muokataan tätä sitten tarvittaessa.